internal/lookoutingesterv2/instructions/instructions_test.go

```diff
@@ -675,33 +701,6 @@ func TestTruncatesStringsThatAreTooLong(t *testing.T) {
 	assert.Len(t, *actual.JobRunsToUpdate[0].Node, 512)
 }
 
-func TestAnnotations(t *testing.T) {
-	annotations := map[string]string{userAnnotationPrefix + "a": "b", "1": "2"}
-	expected := []*model.CreateUserAnnotationInstruction{
-		{
-			JobId:  testfixtures.JobIdString,
-			Key:    "1",
-			Value:  "2",
-			Queue:  testfixtures.Queue,
-			Jobset: testfixtures.JobSetName,
-		},
-		{
-			JobId:  testfixtures.JobIdString,
-			Key:    "a",
-			Value:  "b",
-			Queue:  testfixtures.Queue,
-			Jobset: testfixtures.JobSetName,
-		},
-	}
-	instructions := createUserAnnotationInstructions(
-		testfixtures.JobIdString,
-		testfixtures.Queue,
-		testfixtures.JobSetName,
-		extractUserAnnotations(userAnnotationPrefix, annotations),
-	)
```

Comment:
I think that we should just get rid of this test, because it reimplements part of `handleSubmitJob` (which means that it's bound to diverge from the actual code in production, just like it did here).